testing/tests/headless.test.js

```diff
@@ -20,7 +20,9 @@ describe("Basic viewing of functional pages", () => {
   it("open the /en-US/docs/Web/Foo page", async () => {
     await page.goto(testURL("/en-US/docs/Web/Foo"));
     await expect(page).toMatch("<foo>: A test tag");
-    await expect(page).toMatchElement(".document-meta time", { visible: true });
+    await expect(page).toMatchElement(".last-modified-date time", {
```

Comment:
I think that should be 
```suggestion
    await expect(page).toMatchElement(".metadata time", {
```
because it's more general and it says that the `time` attribute is part of the metadata region independent of the classname of some detail within.